tests/licensedcode/data/plugin_license/scan/ffmpeg-license.expected.json

```diff
@@ -1,4 +1,41 @@
 {
+  "headers": [
```

Comment:
The header portion is not required, we remove this to reduce test expectation updates.